glossary.yml

```diff
@@ -1155,6 +1155,14 @@
       Ni thamani halisi inayopitishwa. Vigezo na hoja ni tofauti, lakini dhana zinazohusiana. 
       Vigezo ni vigeuzo na hoja ni thamani zilizopewa vigeu hivyo.
 
+  uk:
+    term: "аргумент"
+    def: >
+      Термін не слід плутати з [параметром](#parameter), це не синонім. Аргумент — це одне з можливих 
```

Comment:
@Dod900ls1 I think this matches the guidelines from contributor's guide:
> Translate the term in square brackets and leave the text in round brackets in English.

Perhaps while there is no translation for "parameter", the hyperlink will just resolve to the English version? I guess we will wait to see...